src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -24,86 +25,92 @@
 import games.strategy.util.Tuple;
 
 public class UnitsDrawer implements IDrawable {
-  private final int m_count;
-  private final String m_unitType;
-  private final String m_playerName;
-  private final Point m_placementPoint;
-  private final int m_damaged;
-  private final int m_bombingUnitDamage;
-  private final boolean m_disabled;
-  private final boolean m_overflow;
-  private final String m_territoryName;
-  private final IUIContext m_uiContext;
-
+  private final int count;
+  private final String unitType;
+  private final String playerName;
+  private final Point placementPoint;
+  private final int damaged;
+  private final int bombingUnitDamage;
+  private final boolean disabled;
+  private final boolean overflow;
+  private final String territoryName;
+  private final IUIContext uiContext;
+  private final static List<String> STATIC_UNITS = Arrays.asList("aaGun","factory");//TODO add every "static" non-movable unit type of every map.
+  private static boolean drawUnitNation = true;
   public UnitsDrawer(final int count, final String unitType, final String playerName, final Point placementPoint,
       final int damaged, final int bombingUnitDamage, final boolean disabled, final boolean overflow,
       final String territoryName, final IUIContext uiContext2) {
-    m_count = count;
-    m_unitType = unitType;
-    m_playerName = playerName;
-    m_placementPoint = placementPoint;
-    m_damaged = damaged;
-    m_bombingUnitDamage = bombingUnitDamage;
-    m_disabled = disabled;
-    m_overflow = overflow;
-    m_territoryName = territoryName;
-    m_uiContext = uiContext2;
+    this.count = count;
```

Comment:
liking the variable rename. Given we are jumping to a new major version, it is a rare time where it is 100% safe (serialization and reflection make simple variable renames a headache sometimes)